libs/langgraph/langgraph/pregel/__init__.py

```diff
@@ -1151,22 +1152,22 @@ async def aget_state_history(
                 checkpoint_tuple.config, checkpoint_tuple
             )
 
-    def update_state(
+    def bulk_update_state(
         self,
         config: RunnableConfig,
-        values: Optional[Union[dict[str, Any], Any]],
-        as_node: Optional[str] = None,
+        updates: list[BulkUpdate],
     ) -> RunnableConfig:
-        """Update the state of the graph with the given values, as if they came from
-        node `as_node`. If `as_node` is not provided, it will be set to the last node
-        that updated the state, if not ambiguous.
-        """
+        """Apply updates to the graph state in bulk. Requires a checkpointer to be set."""
```

Comment:
could we document the API (all of the params) - i think it's useful to know what the `updates` are just from reading the method docstring